crowdsec-docs/unversioned/beta_program.mdx

```diff
@@ -0,0 +1,56 @@
+---
+id: beta_program
+title: Beta Program
+sidebar_position: 1
+---
+<p align="center">
+<img src='/img/beta_program_header.png' alt="CrowdSec Beta Program" title="CrowdSec Beta Program" width="100%" />
+</p>
+
+## About the Beta Program
+
+Join our Beta program to get early access to new features and functionalities for the CrowdSec Security Engine and Console. Your participation and feedback are key to CrowdSec’s community-driven approach. 
+
+Help us refine and improve our offerings and provide solutions that serve you and the community better. 
+To join the CrowdSec Beta program, click the [Beta opt-in option directly in the Console](https://app.crowdsec.net/settings).
+
+<p align="center">
+<img src='/img/console_beta_program_optin_section.png' alt="CrowdSec Beta Program" title="CrowdSec Beta Program" width="75%" />
+</p>
+
+## Upcoming Betas
+
+### CrowdSec AppSec component - Beta
+
+#### What is it ?
+
+This upcoming new features on some of our front facing components is meant to enhance your defense against malicious activities.
+The AppSec capabilities, primarily aimed at virtual patching, will allow you to turn your CrowdSec install into a full fledged WAF (Web Application Firewall)
+
+To achieve this, the component will channel request data through the behavior engine, broadening vulnerability detection to encompass the full range of exploits via ingress traffic. 
+Alongside our comprehensive collection of WAF rules, we offer the flexibility to craft your own using our intuitive and robust rule language/model. This feature empowers you to customize your WAF to fit your particular security needs.
+
+This feature will integrate seamlessly with our existing web servers and proxy Remediation Components, including Nginx, HAProxy, Traefik Proxy, Apache, and more. 
+
+**Important note**: During the beta phase, this feature is exclusively available for the Nginx Remediation Component. (*link coming soon, when beta is out !*)
+
+#### Quick install guide
+
+To begin exploring the capabilities of our AppSec enabled components, follow our straightforward installation guide:
+*Quick install guide coming soon, when beta is out !*
+
+#### Things you should try out
+
+Don’t know where to start? Here’s a list of a few cool things you can do to turn CrowdSec into a WAF. 
+Give it a try and don’t forget to share your feedback!
+
+- Install wapp-rules/configs
```

Comment:
appsec-rules

Reply:
haha my typo hid it from my search and replace :D